sdk/lib/atl/atlimage.h

```diff
@@ -374,7 +374,11 @@ class CImage
         return m_hbm == NULL;
     }
 
+#ifdef MSPAINT
+    HRESULT Load(LPCTSTR pszFileName, float *pxDpi = NULL, float *pyDpi = NULL) throw()
```

Comment:
No, please revert those ATL changes, as you are breaking compatibility with MS.
Either you are able to retrieve the information needed with the existing functions defined there
(or add any missing function that also exist in the official ATL, see https://learn.microsoft.com/en-us/cpp/atl-mfc-shared/reference/cimage-class?view=msvc-170 ), or, you call those extra functions e.g. the GetImageHorizontalResolution() in mspaint itself to retrieve the missing information. Either directly (just after you call the Load/Save methods of CImage in mspaint), or by defining a class derived from CImage, within mspaint, that adds the extra functionality.

Reply:
I added `CImageDx` that extends `CImage`.
Fixed.